README.md

```diff
@@ -181,11 +175,10 @@ You can deploy this project using:
 
   >
 
-👤 **Author2**
+👤 **Haben Mehari**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@habitoz](https://github.com/habitoz)
+- Linkedin: [@habitoz](https://www.linkedin.com/in/haben-mehari-593531140)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- 🙏Please remove the placeholder text in the Acknowledgements👍 section and provide your custom description. The current text is not customized. The purpose of the placeholder text was to provide guidance on what to do in the section below.
![image](https://github.com/Yonashenok/Awesome-books/assets/63445083/9467fd4f-ed3c-4a95-9c3e-7f15e8f93a27)